crates/ruff_python_formatter/src/expression/operator.rs

```diff
@@ -0,0 +1,44 @@
+use crate::prelude::*;
+use ruff_formatter::{FormatOwnedWithRule, FormatRefWithRule};
+use ruff_python_ast::Operator;
+
+#[derive(Copy, Clone)]
+pub struct FormatOperator;
```

Comment:
I moved this out of the `bin_expr` formatting because it is also used in augmented assign and it removes the size of the `bin_expr` file a bit. The code itself is unchanged.